.swift-format

```diff
@@ -0,0 +1,11 @@
+{
+    "version": 1,
+    "lineLength": 10000,
+    "indentation": {
```

Comment:
suggestion: I'm haven't read the documentation ,but based on the setting name, we should consider `"multiElementCollectionTrailingCommas" : true,`.

Reply:
That corresponds to:

```swift
[
    "a",
    "b"
]
```

vs

```swift
[
    "a",
    "b",
]
```

Support for the latter was added to Swift some time ago, the argument being that it helps reduce the SCM diff as new lines are added to the bottom.

I'd be fine with enabling that rule.

Reply:
I want the latter :)